fiftyone/core/labels.py

```diff
@@ -401,18 +401,118 @@ class Detection(_HasAttributesDict, _HasID, Label):
         mask (None): an instance segmentation mask for the detection within
             its bounding box, which should be a 2D binary or 0/1 integer numpy
             array
+        mask_path (None):  the absolute path to the instance segmentation image
+            on disk
         confidence (None): a confidence in ``[0, 1]`` for the detection
         index (None): an index for the object
         attributes ({}): a dict mapping attribute names to :class:`Attribute`
             instances
     """
 
+    _MEDIA_FIELD = "mask_path"
```

Comment:
Note to Voxel51, this would be the first instance of a media field that could be embedded in a list (`Detections`), which means these methods will need to be updated:
- https://github.com/voxel51/fiftyone/blob/1824ddb5f579842664b9caf2f400ab2934191ff1/fiftyone/utils/data/exporters.py#L2345
- https://github.com/voxel51/fiftyone/blob/1824ddb5f579842664b9caf2f400ab2934191ff1/fiftyone/utils/data/exporters.py#L2041

There could be more such places too.